tasks/models.py

```diff
@@ -25,6 +26,15 @@ class Task(models.Model):
     status = EnumChoiceField(Status, default=Status.BACKLOG, max_length=1)
     description = models.TextField(null=True, blank=True)
 
+    @classmethod
+    def update_status(task_id, status):
```

Comment:
Since it is a class method, add cls as the first argument.
def update_status(cls, task_id, status)
Otherwise the function won't work properly

Reply:
I would go with `@staticmethod`